lib/i18n/js/formatters/js.rb

```diff
@@ -1,5 +1,10 @@
 require "i18n/js/formatters/base"
 
+JSON_ESCAPE_MAP = {
+  "'" => "\\'",
+  "\\" => "\\\\"
+}
```

Comment:
1. This hash should be frozen
2. This constant should be placed inside the class so that it does not affect top namespace
3. Good to have `private_constant : JSON_ESCAPE_MAP` after it's moved into class
https://docs.rubocop.org/rubocop/1.0/cops_style.html#styleconstantvisibility